woocommerce/woocommerce-android
# [Simplified login] Update illustrations for native Jetpack installation and site picker

Closes: #7916 

⚠️ Please don't merge before #7942 

### Description
This PR updates the following illustrations:
1. Site picker empty state.
2. Jetpack Installation initial screen.
3. Jetpack Connection initial screen.
4. Jetpack Connection interruption screen.

The site picker's illustration doesn't have a dark them version yet, I left a question about it here p1669273884341549/1668447780.403299-slack-C046HDZL87J 

Also, I'm following the same approach as iOS, I updated only the illustration and the text, I didn't make any changes to the header. There is an updated version of this screen as part of the store creation project, so it's not worth the double effort.

### Testing instructions

Please make sure the Jetpack Native experiment is used, you can either hardcode the value in JetpackInstallationExperiment or set your device as test device in Firebase Console.

##### Site Picker
Connect to the app using an account that's not connected to any stores.

##### Install Jetpack
1. Open the site picker.
5. Click on Add a store, then select "Connect an existing store"
6. Enter address of a non-jetpack site.
7. Click on Continue.
8. Confirm the illustration for "Install Jetpack"

##### Connect Jetpack and dismissing connection
1. Open the site picker.
2. Click on Add a store, then select "Connect an existing store"
3. Enter the address of a site that has Jetpack but it's not connected.
4. Click on Continue.
9. Confirm the illustration for "Connect Jetpack"
10. Click on Connect Jetpack.
11. Enter site credentials, then click on Connect Jetpack.
12. When the WebView opens, close it.
13. Confirm the illustration for Connection Dismissed

### Images/gif
| | Light | Dark |
| --- | --- | --- |
| Site Picker | <img width="436" alt="Screen Shot 2022-11-24 at 17 53 29" src="https://user-images.githubusercontent.com/1657201/203835169-ae9cd0f7-6032-4130-b275-7dfd18a6531d.png"> | <img width="436" alt="Screen Shot 2022-11-24 at 17 53 44" src="https://user-images.githubusercontent.com/1657201/203835209-9220ff99-1352-4909-8069-53e080768fc2.png"> |
| Install Jetpack | <img width="436" alt="Screen Shot 2022-11-24 at 18 00 20" src="https://user-images.githubusercontent.com/1657201/203836213-fb02e718-449a-4f11-9c0e-857529146dfc.png"> | <img width="436" alt="Screen Shot 2022-11-24 at 18 00 10" src="https://user-images.githubusercontent.com/1657201/203836192-3d7df8bf-c4bf-4616-a53f-54524a34ad15.png"> |
| Connect Jetpack | <img width="436" alt="Screen Shot 2022-11-24 at 17 58 12" src="https://user-images.githubusercontent.com/1657201/203835892-67b42100-3aed-4cc7-b2ae-c44495c3f20b.png"> | <img width="436" alt="Screen Shot 2022-11-24 at 17 57 23" src="https://user-images.githubusercontent.com/1657201/203835753-f147a7b4-143e-43e5-8ccd-f66f166619f8.png"> | 
| Connection Dismissed | <img width="436" alt="Screen Shot 2022-11-24 at 17 59 00" src="https://user-images.githubusercontent.com/1657201/203836049-8d4bb03f-d5ed-4386-aa0a-2c82db577d59.png"> | <img width="436" alt="Screen Shot 2022-11-24 at 17 59 36" src="https://user-images.githubusercontent.com/1657201/203836095-fdacec33-bdbd-4c4e-b5d0-536a0778dc96.png"> |
